client/src/main/java/com/influxdb/client/BucketsQuery.java

```diff
@@ -0,0 +1,112 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class BucketsQuery {
+
+    /**
+     * offset  (optional)
+     */
+    @Nullable
+    private Integer offset;
+
+    /**
+     *  (optional, default to 20)
+     */
+    @Nullable
+    private Integer limit;
+
+    /**
+     * The last resource ID from which to seek from (but not including). This is to be used instead of &#x60;offset&#x60;. (optional)
+     */
+    @Nullable
+    private String after;
+
+    /**
+     * The name of the organization. (optional)
+     */
+    @Nullable
+    private String org;
+
+    /**
+     * The organization ID. (optional)
+     */
+    @Nullable
+    private String orgID;
+
+    /**
+     * Only returns buckets with a specific name. (optional)
+     */
+    @Nullable
+    private String name;
+
+    /**
+     * Only returns buckets with a specific ID. (optional)
+     */
+    @Nullable
+    private String id;
+
+    @Nullable
+    public Integer getOffset() {
+        return offset;
+    }
+
+    public void setOffset(@Nullable Integer offset) {
+        this.offset = offset;
+    }
+
+    @Nullable
+    public Integer getLimit() {
+        return limit;
+    }
+
+    public void setLimit(@Nullable Integer limit) {
+        this.limit = limit;
+    }
+
+    @Nullable
+    public String getAfter() {
+        return after;
+    }
+
+    public void setAfter(@Nullable String after) {
```

Comment:
Add `final` modifier:
```suggestion
    public void setAfter(@Nullable final String after) {
```